rosalind_hamm.py

```diff
@@ -0,0 +1,9 @@
+def main(data):
+    a, b = data[:-1].split('\n')
```

Comment:
The reason I have the `[:-1]` is because there was an extra "\n" at the end of the file, which caused `.split(\n)` to want to unpack 3 variable. So I just made it split the characters from  0 to -1 of `data`.